spec.bs

```diff
@@ -2563,6 +2573,13 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
     set  [=this=]'s [=relevant global object=]'s
       [=InterestGroupReportingScriptRunnerGlobalScope/report url=] to failure
       and [=exception/Throw=] a {{TypeError}}.
+  1. Optionally, set  [=this=]'s [=relevant global object=]'s
+      [=InterestGroupReportingScriptRunnerGlobalScope/report url=] to failure
+      and [=exception/Throw=] a {{TypeError}}.

```

Comment:
Any reason this is `TypeError` here and not `NotAllowedError` as the two above?

Reply:
Let me add @caraitto to confirm the exception type in our implementation for sendReportTo enrollment checks.

I had followed the existing exception type for sendReportTo, e.g.:
"If parsedUrl is failure, or parsedUrl’s [scheme](https://url.spec.whatwg.org/#concept-url-scheme) is not "https", ... [Throw](https://webidl.spec.whatwg.org/#dfn-throw) a [TypeError](https://webidl.spec.whatwg.org/#exceptiondef-typeerror)."

Reply:
No exception is thrown -- instead, the reporting URL is erased from the set of URLs to report [0], and an error is logged to devtools [1][2].

[0] https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_auction_reporter.cc;l=1031;drc=eef4762779d05708d5dfc7d5fe4ea16288069a35

[1] https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_auction_reporter.cc;l=1011;drc=4f1dab1d89ebceedb5f01646174822fedb9ff7e8

[2] https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/ad_auction_service_impl.cc;l=784;drc=eef4762779d05708d5dfc7d5fe4ea16288069a35

Reply:
Thanks! updated to return w/o an exception.

Reply:
Cool, thanks.

Reply:
Just curious, does devtools logging show up in specs?

Reply:
> Just curious, does devtools logging show up in specs?

It does not.